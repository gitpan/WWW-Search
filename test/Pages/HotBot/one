http://www.hotbot.com/text/default.asp?SM=SC&MT=LSAM+replication&DE=2&DC=100
