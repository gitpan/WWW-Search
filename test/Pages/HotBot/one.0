
<html>
<head><title>HotBot results: LSAM replication (1+)</title></head>
<body>
<br>
<form action="/text/default.asp">
<font face=Arial,Geneva,sans-serif>&nbsp;&nbsp;&nbsp;<b>HotBot: Text-only version</b> | 
<a href="/?act.revise=+Revise+&MT=LSAM+replication&SM=SC&DC=100">Full graphics version</a>
<P>
</font>
<font face="Arial, Helvetica" size="2">
<p>
&nbsp;&nbsp;&nbsp;<B>look for:</b>&nbsp;<SELECT name="SM" >
<OPTION value="MC">all the words
<OPTION value="SC" selected>any of the words
<OPTION value="phrase">exact phrase
<OPTION value="title">the page title
<OPTION value="name">the person
<OPTION value="url">links to this URL
<OPTION value="B">Boolean phrase
</SELECT>
&nbsp;<INPUT type=text name="MT" value="LSAM replication" size=30 maxlength=300>
&nbsp;<input type="submit" name="search" value="Search"><br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<input type="SUBMIT" name="act.new" value=" New ">&nbsp;<input type="SUBMIT" name="act.revise" value=" Revise ">&nbsp;<input type="SUBMIT" name="act.super" value=" More Options ">
<br>
</font>
<font face="Arial, Helvetica" font size="1">

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<input TYPE=Checkbox NAME="SWR" VALUE="LSAM replication" >
<i>Search within these results.</i><p>


 
</font>
<br>&nbsp;<!-- adSpaceCOM --><!-- AdSpace program=hotbotresults size=large SERVER_NAME=www.hotbot.com KW=lsam KW=replication KW=lsam%20replication KW=lsam%20replication ALL_KW=lsam%20replication erlicat=nothing// section=erli/nothing// cobrand= REMOTE_HOST=copper.dulles.tasc.com RH1=com RH2=tasc.com RH3=dulles.tasc.com RH4=copper.dulles.tasc.com REMOTE_ADDR=147.81.3.197 R_UNIQID=3soJvcZpxGQD P_UNIQID=3soJvcZpxGgD hbf=adult hbf=language:any --><a href="http://nsads.hotwired.com/event.ng/Type=click&ProfileID=4923&RunID=7408&AdID=10405&GroupID=1&FamilyID=618&TagValues=2.5.6.25.145.156.174.186.258.374.389.409.526.539.6456.6626.6820.6830.72561.75378.75606&amp;params.tag.ALL_KW=lsam+replication&Redirect=http:%2F%2Fshop.hotbot.com">Try HotBot's Real Estate Center!<br><img src="http://static.wired.com/advertising/blipverts/houseads/shop_loan.gif" border=1 height=60 width=468 alt="Try HotBot's Real Estate Center!"></a><!-- /AdSpace --><!-- /adSpaceCOM -->


<blockquote>
<b>Returned:&nbsp;16&nbsp;Matches&nbsp;
<br>Results for &quot;LSAM replication&quot;</b><br>


<br>
<font face="Verdana, Arial, Helvtica" size="1">

	
	<b>1&nbsp;-&nbsp;16&nbsp;</b><br>
<br>

<!-- BRES --> 

<p> <b>1.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fwww%2Eisi%2Eedu%2Flsam%2Fabout%2Freplication%2Ehtml&id=1&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>About LSAM Replication</A></b><br>About LSAM Replication Why Replication? Although the Web is widely used today, for it to be truly effective part of daily work, it must provide both dependable availability and performance. Today, the Web often suffers in both these  categories..<br><font size=1><b>99%&nbsp&nbsp</b><i>1/7/97</i>  http://www.isi.edu/lsam/about/replication.html<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fwww%2Eisi%2Eedu%2Flsam%2Fabout%2Freplication%2Ehtml">this site only</a>.</font><p> <b>2.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fnii%2Eisi%2Eedu%2Fpeople%2Fjohnh%2FWORK%2Findex%2Ehtml&id=2&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>John Heidemann's Work</A></b><br>John Heidemann's Work I am computer scientist at Information Sciences Institute of the University of Southern California. I will also be teaching at USC in Spring semester 1997. See LA through ISI's eyes (or at least windows). ISI At ISI  I'm..<br><font size=1><b>95%&nbsp&nbsp</b><i>6/26/97</i>  http://nii.isi.edu/people/johnh/WORK/index.html<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fnii%2Eisi%2Eedu%2Fpeople%2Fjohnh%2FWORK%2Findex%2Ehtml">this site only</a>.</font><p> <b>3.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fgost%2Eisi%2Eedu%2F%7Ejohnh%2FWORK%2Findex%2Ehtml&id=3&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>John Heidemann's Work</A></b><br>John Heidemann's Work I am computer scientist at Information Sciences Institute and a research assistant professor at the University of Southern California's Computer Science Department. See LA through ISI's eyes (or at least windows). ISI At ISI  I<br><font size=1><b>95%&nbsp&nbsp</b><i>11/20/98</i>  http://gost.isi.edu/~johnh/WORK/index.html<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fgost%2Eisi%2Eedu%2F%7Ejohnh%2FWORK%2Findex%2Ehtml">this site only</a>.</font><p> <b>4.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fsphere%2Ecs%2Ecaltech%2Eedu%2Fmailing%5Flists%2Fdist%2Dobj%2Fmsg00886%2Ehtml&id=4&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>[Fwd: LSAM: Large-Scale Active Middleware (formerly Cities Online)]</A></b><br>[Date Prev][Date Next][Thread Prev][Thread Next][Date Index][Thread Index] [Fwd: LSAM: Large-Scale Active Middleware (formerly Cities Online)] To: dist-obj@cs.caltech.edu Subject: [Fwd: LSAM: Large-Scale Active Middleware (formerly Cities  Online).<br><font size=1><b>94%&nbsp&nbsp</b><i>3/4/99</i>  http://sphere.cs.caltech.edu/mailing_lists/dist-obj/msg00886.html<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fsphere%2Ecs%2Ecaltech%2Eedu%2Fmailing%5Flists%2Fdist%2Dobj%2Fmsg00886%2Ehtml">this site only</a>.</font><p> <b>5.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fwww%2Eresearch%2Eatt%2Ecom%2F%7Emisha%2Ftutorial%2Ftutorial%2Dbib%2Etxt&id=5&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>http://www.research.att.com/~misha/tutorial/tutorial-bib.txt</A></b><br>%NOTE: This is not intended to be an inlusive bibliography list. %It only represents limited works from the areas I chose to emphasize %in my tutorial. Also, this list does not include references to %commercial products discussed in the  tutorial..<br><font size=1><b>90%&nbsp&nbsp</b><i>10/1/98</i>  http://www.research.att.com/~misha/tutorial/tutorial-bib.txt<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fwww%2Eresearch%2Eatt%2Ecom%2F%7Emisha%2Ftutorial%2Ftutorial%2Dbib%2Etxt">this site only</a>.</font><p> <b>6.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fwww%2Enetpromote%2Ecom%2Fsearch%2FWWW%2DSearch%2D1%2E023%2Fblib%2Fman3%2FWWW%3A%3ASearch%3A%3ANorthernLight%2E3&id=6&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>http://www.netpromote.com/search/WWW-Search-1.023/blib/man3/WWW::Search::NorthernLight.3</A></b><br>.rn '' }` $RCSfile$$Revision$$Date$ '' $Log$ '' .de Sh .br .if t .Sp .ne 5 .PP \fB\\$1\fR .PP ..de Sp .if t .sp .5v .if n .sp ..de Ip .br .ie \\n(.$&gt;=3 .ne \\$3 .el .ne 3 .IP &quot;\\$1&quot; \\$2 ..de Vb .ft CW .nf .ne \\$1 ..de Ve .ft R .fi .. '' Set up  .<br><font size=1><b>90%&nbsp&nbsp</b><i>3/5/99</i>  http://www.netpromote.com/search/WWW-Search-1.023/blib/man3/WWW::Search::NorthernLight.3<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fwww%2Enetpromote%2Ecom%2Fsearch%2FWWW%2DSearch%2D1%2E023%2Fblib%2Fman3%2FWWW%3A%3ASearch%3A%3ANorthernLight%2E3">this site only</a>.</font><p> <b>7.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fwwwcache%2Eja%2Enet%2Fevents%2Fworkshop%2F14%2Flsam%2Ehtml&id=7&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>Abstract</A></b><br>The LSAM Proxy Cache - a Multicast Distributed Virtual Cache Joe Touch USC / Information Sciences Institute touch@isi.edu (PDF print-formatted version of this file) Abstract 1` The LSAM Proxy is a multicast distributed web cache that  provides..<br><font size=1><b>86%&nbsp&nbsp</b><i>4/14/98</i>  http://wwwcache.ja.net/events/workshop/14/lsam.html<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fwwwcache%2Eja%2Enet%2Fevents%2Fworkshop%2F14%2Flsam%2Ehtml">this site only</a>.</font><p> <b>8.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fwww%2Escimitar%2Eterena%2Enl%2Fstandardisation%2Freports%2FIETF43%5FPalme%2FIETF43%5FPalme%2Ehtml&id=8&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>IETF December 1998 Meeting Notes</A></b><br>&quot;&gt; Applications IETF December 1998 Notes Personal notes from mainly applications area sessions during the IETF meeting in Orlando, December 1998 By Professor Jacob Palme, Stockholm University and KTH Technical University Abstract Here are  my..<br><font size=1><b>85%&nbsp&nbsp</b><i>6/17/99</i>  http://www.scimitar.terena.nl/standardisation/reports/IETF43_Palme/IETF43_Palme.html<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fwww%2Escimitar%2Eterena%2Enl%2Fstandardisation%2Freports%2FIETF43%5FPalme%2FIETF43%5FPalme%2Ehtml">this site only</a>.</font><p> <b>9.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fathos%2Erutgers%2Eedu%2F%7Edavison%2Fweb%2Dcaching%2Fbibliography%2Ehtml&id=9&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>Brian Davison's Web-Caching Bibliography</A></b><br>Brian Davison's Web-Caching  Bibliograph<br><font size=1><b>80%&nbsp&nbsp</b><i>4/7/99</i>  http://athos.rutgers.edu/~davison/web-caching/bibliography.html<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fathos%2Erutgers%2Eedu%2F%7Edavison%2Fweb%2Dcaching%2Fbibliography%2Ehtml">this site only</a>.</font><p> <b>10.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fwww%2Elaw%2Eumich%2Eedu%2Fmplp%2Fcasemgmtworkplan%2Ehtm&id=10&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>Case Management Pilot Project Work Plan</A></b><br>Taskforces Training Brief Bank Issue Alerts Poverty Law Materials Legislative Update Jobs Links MPLP Info Desk Reference Technology MPLP Home Case Management Pilot Project Work Plan August 24, 1998 Needs/Goals Case management and  timekeeping..<br><font size=1><b>77%&nbsp&nbsp</b><i>4/18/99</i>  http://www.law.umich.edu/mplp/casemgmtworkplan.htm<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fwww%2Elaw%2Eumich%2Eedu%2Fmplp%2Fcasemgmtworkplan%2Ehtm">this site only</a>.</font><p> <b>11.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fwww%2Emplp%2Eorg%2Fcasemgmtworkplan%2Ehtm&id=11&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>Case Management Pilot Project Work Plan</A></b><br>Taskforces Training Brief Bank Issue Alerts Poverty Law Materials Legislative Update Jobs Links MPLP Info Desk Reference Technology MPLP Home Case Management Pilot Project Work Plan August 24, 1998 Needs/Goals Case management and  timekeeping..<br><font size=1><b>77%&nbsp&nbsp</b><i>4/18/99</i>  http://www.mplp.org/casemgmtworkplan.htm<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fwww%2Emplp%2Eorg%2Fcasemgmtworkplan%2Ehtm">this site only</a>.</font><p> <b>12.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fwww%2Dsor%2Einria%2Efr%2Fprojects%2Frelais%2Fbiblio%2F&id=12&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>Web caching bibliography</A></b><br>Web caching bibliography This bibliography is also available in BibTeX and gzipped postscript formats. You can also search the bibliography thanks to the Collection of Computer Science Bibliographies. See also: The Relais home page The  WebTech..<br><font size=1><b>77%&nbsp&nbsp</b><i>5/23/99</i>  http://www-sor.inria.fr/projects/relais/biblio/<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fwww%2Dsor%2Einria%2Efr%2Fprojects%2Frelais%2Fbiblio%2F">this site only</a>.</font><p> <b>13.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fnetresearch%2Eics%2Euci%2Eedu%2Fresearchclass%2Fweb%2Ehtml&id=13&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>Web caching bibliography</A></b><br>Web caching bibliography This bibliography is also available in BibTeX and gzipped postscript formats. You can also search the bibliography thanks to the Collection of Computer Science Bibliographies. See also: The Relais home page The  WebTech..<br><font size=1><b>77%&nbsp&nbsp</b><i>5/1/99</i>  http://netresearch.ics.uci.edu/researchclass/web.html<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fnetresearch%2Eics%2Euci%2Eedu%2Fresearchclass%2Fweb%2Ehtml">this site only</a>.</font><p> <b>14.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fwww%2Ecs%2Ewpi%2Eedu%2F%7Ewebbib%2Fwebbib%2Ddate%2Ehtml&id=14&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>Core set of bibliographic references (Sorted by Last Modification Time)</A></b><br>bibliography<br><font size=1><b>72%&nbsp&nbsp</b><i>4/15/99</i>  http://www.cs.wpi.edu/~webbib/webbib-date.html<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fwww%2Ecs%2Ewpi%2Eedu%2F%7Ewebbib%2Fwebbib%2Ddate%2Ehtml">this site only</a>.</font><p> <b>15.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fwww%2Efreecode%2Ecom%2Fcgi%2Dbin%2Fviewsource%2Epl&id=15&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>FreeCode, Free Internet Source Code</A></b><br>The FreeCode website is the world's largest collection of free Internet related source code. Free programs are available in the following languages: C/C  , Java, Perl, and Visual Basic. <br><font size=1><b>65%&nbsp&nbsp</b><i>6/17/99</i>  http://www.freecode.com/cgi-bin/viewsource.pl<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fwww%2Efreecode%2Ecom%2Fcgi%2Dbin%2Fviewsource%2Epl">this site only</a>.</font><p> <b>16.&nbsp;&nbsp;<a href=/director.asp?target=http%3A%2F%2Fwww%2Ekrnic%2Enet%2Fftpsite%2Finternet%2Ddrafts%2F1id%2Dabstracts%2Etxt&id=16&userid=3soJvcZpxGgD&query=MT=LSAM+replication&SM=SC&DC=100&rsource=INK>http://www.krnic.net/ftpsite/internet-drafts/1id-abstracts.txt</A></b><br>Current Internet-Drafts This summary sheet provides a short synopsis of each Internet-Draft available within the &quot;internet-drafts&quot; directory at the shadow sites directory. These drafts are listed alphabetically by working group acronym and  initial.<br><font size=1><b>61%&nbsp&nbsp</b><i>2/9/97</i>  http://www.krnic.net/ftpsite/internet-drafts/1id-abstracts.txt<BR>See results from <a href="/?MT=LSAM+replication&SM=SC&DC=100&uHost=http%3A%2F%2Fwww%2Ekrnic%2Enet%2Fftpsite%2Finternet%2Ddrafts%2F1id%2Dabstracts%2Etxt">this site only</a>.</font><p>
<INPUT type=hidden name="DC" value="100" >

<!-- ERES -->

<INPUT type=hidden name="_v" value="2" >
<INPUT type=hidden name="OPs" value="MDRTP" >

</form>

	
	<b>1&nbsp;-&nbsp;16&nbsp;</b><p>

<!-- hotbotMon:success -->
<br>
<a href="/addurl.asp">Add URL</a> | <a href="mailto:wow@wired.com">Send&nbsp;us&nbsp;feedback</a>&nbsp;| 
<a href="http://www.hotwired.com/jobs/">Work&nbsp;at&nbsp;HotBot</a>&nbsp;| <a href="http://www.wired.com/home/advertising/">Advertise&nbsp;on&nbsp;HotBot</a>&nbsp;| 
<a href="http://www.wired.com/home/">Company&nbsp;Information</a>&nbsp;| 
<a href="http://www.wired.com/home/digital/privacy/">Our&nbsp;Privacy&nbsp;Policy</a>&nbsp;|
<a href="/help/">Help</a>&nbsp;| 
<a href="/text/"><nobr>Text-only version</nobr></a> 
<p>
<nobr><a href="http://www.wired.com/home/copyright.html">Copyright</a> &copy; 1994-99 Wired Digital Inc. All rights reserved.<br></nobr>

</font>
</blockquote>
</body>
</html>

