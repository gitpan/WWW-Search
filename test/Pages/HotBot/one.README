This set of test-result pages was created on Tue Sep 28 13:36:07 1999
