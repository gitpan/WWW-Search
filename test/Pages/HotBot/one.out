http://www.isi.edu/lsam/about/replication.html
http://nii.isi.edu/people/johnh/WORK/index.html
http://gost.isi.edu/~johnh/WORK/index.html
http://sphere.cs.caltech.edu/mailing_lists/dist-obj/msg00886.html
http://www.research.att.com/~misha/tutorial/tutorial-bib.txt
http://www.netpromote.com/search/WWW-Search-1.023/blib/man3/WWW::Search::NorthernLight.3
http://wwwcache.ja.net/events/workshop/14/lsam.html
http://www.scimitar.terena.nl/standardisation/reports/IETF43_Palme/IETF43_Palme.html
http://athos.rutgers.edu/~davison/web-caching/bibliography.html
http://www.law.umich.edu/mplp/casemgmtworkplan.htm
http://www.mplp.org/casemgmtworkplan.htm
http://www-sor.inria.fr/projects/relais/biblio/
http://netresearch.ics.uci.edu/researchclass/web.html
http://www.cs.wpi.edu/~webbib/webbib-date.html
http://www.freecode.com/cgi-bin/viewsource.pl
http://www.krnic.net/ftpsite/internet-drafts/1id-abstracts.txt
